<?xml version="1.0" encoding="utf-8" standalone="no"?>
<!--This file represents the results of running a test suite-->
<test-results name="C:\src\PersonalServiceBus\PersonalServiceBus.RSS.Test.Unit\bin\Debug\PersonalServiceBus.RSS.Test.Unit.dll" total="1" errors="1" failures="0" not-run="0" inconclusive="0" ignored="0" skipped="0" invalid="0" date="2013-05-27" time="20:55:27">
  <environment nunit-version="2.6.2.12296" clr-version="2.0.50727.5466" os-version="Microsoft Windows NT 6.1.7601 Service Pack 1" platform="Win32NT" cwd="C:\src\PersonalServiceBus" machine-name="ANDERSON" user="jake.anderson" user-domain="TRITON-TEK" />
  <culture-info current-culture="en-US" current-uiculture="en-US" />
  <test-suite type="Assembly" name="C:\src\PersonalServiceBus\PersonalServiceBus.RSS.Test.Unit\bin\Debug\PersonalServiceBus.RSS.Test.Unit.dll" executed="True" result="Failure" success="False" time="1.584" asserts="0">
    <results>
      <test-suite type="Namespace" name="PersonalServiceBus" executed="True" result="Failure" success="False" time="1.555" asserts="0">
        <results>
          <test-suite type="Namespace" name="RSS" executed="True" result="Failure" success="False" time="1.554" asserts="0">
            <results>
              <test-suite type="Namespace" name="Test" executed="True" result="Failure" success="False" time="1.554" asserts="0">
                <results>
                  <test-suite type="Namespace" name="Unit" executed="True" result="Failure" success="False" time="1.554" asserts="0">
                    <results>
                      <test-suite type="TestFixture" name="FeedManagerTest" executed="True" result="Failure" success="False" time="1.549" asserts="0">
                        <results>
                          <test-case name="PersonalServiceBus.RSS.Test.Unit.FeedManagerTest.AddFeedTest" executed="True" result="Error" success="False" time="1.530" asserts="0">
                            <failure>
                              <message><![CDATA[System.NullReferenceException : Object reference not set to an instance of an object.]]></message>
                              <stack-trace><![CDATA[at Raven.Client.DocumentStoreBase.set_Url(String value) in c:\Builds\RavenDB-Stable\Raven.Client.Lightweight\DocumentStoreBase.cs:line 110
at PersonalServiceBus.RSS.Infrastructure.RavenDB.RavenDatabase..ctor(IConfiguration configuration) in c:\src\PersonalServiceBus\PersonalServiceBus.RSS.Infrastructure\RavenDB\RavenDatabase.cs:line 18
at PersonalServiceBus.RSS.Test.Unit.FeedManagerTest.AddFeedTest() in c:\src\PersonalServiceBus\PersonalServiceBus.RSS.Test.Unit\FeedManagerTest.cs:line 18
]]></stack-trace>
                            </failure>
                          </test-case>
                        </results>
                      </test-suite>
                    </results>
                  </test-suite>
                </results>
              </test-suite>
            </results>
          </test-suite>
        </results>
      </test-suite>
    </results>
  </test-suite>
</test-results>